Add instructions for dealing with generic JSON

cparker4486 opened the pull request:
I came to this doc and was thoroughly confused until I realized why Filter array wasn't working as described in this doc.

azurecla commented:
Hi __@kevinscharpenberg__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kevsch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cparker4486 committed:
Add instructions for dealing with generic JSON

I came to this doc and was thoroughly confused until I realized why Filter array wasn't working as described in this doc.

cparker4486 committed:
Implement Jeff's suggested change
